### ISSUE TITLE
Can I ask what this is for?

### ISSUE THREAD

**Issue by tadgh** [arid1995/kaniwani-clicker#1]
Came across this just now.. I'm the author of Kaniwani, and I'm wondering exactly what you're trying to accomplish with this script? Seems to just lock a whole level. Cheers

**Comment by subher0** [comment]
Hi! I, personally, do not have an account on kaniwani, but a friend of mine does. So he, being himself quite a needy fella, asked me for this script. Yes, you guessed right, it just locks/unlocks the words in the level. I, myself, don't know how kaniwani works, but my friend says, that there is no way to lock the whole level, for that he had to manually click on each word. Oh, and it's not finished, as a programmer I'm quite sloppy, so it will take me quite a while to get this done.

**Comment by tadgh** [comment]
ah ok. In case your friend isn't aware, you can lock the entire level (though it deletes progress for that level), by clicking on the icon of the unlocked padlock. 
![](https://i.gyazo.com/f0155bac4a4f63664d5e4f282d4bb0f3.png)

**Comment by subher0** [comment]
Yeah, he knows that, but sometimes he needs to review specific level without losing all his progress. That's why the script automates locking each word in the level.
